etl/steps/data/garden/artificial_intelligence/2023-07-25/cset.meta.yml

```diff
@@ -0,0 +1,161 @@
+    intelligence (AI) globally. This data comes from a vast repository known as the Merged Academic Corpus (MAC), which contains
+    details about more than 270 million academic articles worldwide. In CAT, only those articles that are related to AI are
+    utilized.
+
+
+    CSET determines the country of origin for each article by looking at the affiliations of the authors. If an author is
+    associated with an organization in a specific country, that article is attributed to that country. These articles are
+    then grouped into various AI sub-categories. This categorization is based on a scoring system that assigns scores to each
+    article according to its subject. However, this scoring system only applies to English-language articles. For non-English
+    articles, scores are estimated based on the average scores of articles they either cite or are cited by.
+
+
+    CAT also provides patent data sourced from 1790 Analytics and Dimensions, encompassing AI-related patents. However, instead
+    of indicating patent ownership, CAT highlights where the patents are filed. For instance, a patent filed in the U.S. patent
+    office is not necessarily owned by an American. CSET is striving to include data regarding the nationality of patent owners
+    in future updates of CAT.
+
+
+    Additionally, CAT uses data from Crunchbase to identify AI-related companies and includes metrics about investments in
+    these companies. The definition of an 'AI company' is not straightforward, but CSET identifies such companies based on
+    specific criteria. These include whether the company description contains AI-related keywords, whether Crunchbase tags
+    it as an AI company, or if it is linked with an AI Publication, AI Patent, or a Top AI Conference Publication as per CSET's
+    Private-sector AI-Related Activity Tracker. CSET aims to encompass a broad range of AI-related companies across the globe.
+    Therefore, some companies may be categorized as AI-related that others may not consider as such. Conversely, some companies
+    perceived as AI-related by others may not meet the criteria set by CSET.
+
+
+    CSET ensures the data in CAT is updated at least quarterly, with plans to provide more frequent updates in the future.
+  licenses:
+  - name: Creative Commons BY 4.0
+    url: https://eto.tech/tou/
+  sources:
+  - name: Center for Security and Emerging Technology (2023)
+    url: https://cat.eto.tech/
+    date_accessed: '2023-07-25'
+    publication_year: 2023
+    published_by: Emerging Technology Observatory Country Activity Tracker, Artificial Intelligence (Center for Security and
+      Emerging Technology, 2023)
+tables:
+  cset:
+    variables:
+      disclosed_investment:
+        title: Disclosed investment
+        description:  Investments in AI-related companies across various industries where the investment value has been publicly disclosed.
+        unit: US dollars
```

Comment:
I think these are the variables that have been adjusted for inflation? It would be good to note that you have done this here, following the guide [here](https://docs.google.com/document/d/1tcknTryJAxOR0yv8x38s3jkONq-TY3T4HNCEcZQ_c-s/edit#heading=h.58dyovrzqydc) for the unit.

Reply:
yes, thank you!